test/indexer/test_hca_indexer.py

```diff
@@ -170,7 +170,7 @@ def _assert_old_bundle(self, num_expected_new_contributions=0, ignore_aggregates
             else:
                 self.assertIn('Farmers Trucks', [c.get('institution') for c in project['contributors']])
             specimen = one(contents['specimens'])
-            self.assertIn('Australopithecus', specimen['genus_species'])
```

Comment:
This actually renders the test useless. Fix the input metadata so you don't have to do modify the assertion.